Part3_icon-vis/ex3_0_icon-vis.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8822930b",
+   "metadata": {},
+   "source": [
+    "# Exercise 3.0: icon-vis\n",
+    "prepared by A. Lauber\n",
+    "\n",
+    "In this exercise, we will work with the GitHub repository [icon-vis](https://github.com/C2SM/icon-vis), which was developed by C2SM and MeteoSwiss. It is a collection of easy-to-use Python scripts for plotting data on unstructured grids like the ICON grid. It is based on the library [psyplot](https://psyplot.github.io), which was already introduced in [Exercise 2.2](../Part2_Mapplots/ex2_2_psyplot.ipynb).\n",
+    "\n",
+    "Advantages of plotting on the unstructured grid:\n",
+    "- Shows the real output without interpolation\n",
+    "- No preprocessing of the data necessary\n",
+    "\n",
+    "Disadvantages of plotting on the unstructured grid:\n",
+    "- It can be quite slow depending on the size of your netCDF file\n",
+    "- Not all features are available yet (psyplot is still under development)\n",
+    "\n",
+    "## Installation\n",
+    "\n",
+    "If you want to use icon-vis on your own machine, please follow the [installation instruction](https://github.com/C2SM/icon-vis#environment-setup) in the README. For this course, an environment is already prepared for you to save time."
```

Comment:
Is this the pyvis environment or another? Can you mention this? In https://github.com/C2SM/icon-vis/blob/master/env/environment.yml some stuff is pinned to a very specific version. Could that be relaxed?

Reply:
This is the psyplot environment needed for icon-vis (for people who want to use icon-vis outside this workshop). I added the necessary packages for the exercises here to pyvis.
(Some of the versions in icon-vis are pinned to a very specific version due to conflicts.)